cmd/res/configuration.yaml

```
@@ -0,0 +1,48 @@
+Writable:
+  LogLevel: "INFO"
+
+Service:
+  Port: 59989
+  StartupMsg: "device llrp started"
```

This addition is needed for all services. Not part of YAML change, but still needed.
```suggestion
  Host: localhost
  Port: 59989
  StartupMsg: "device llrp started"
```